server/routes/user.js

```diff
@@ -12,15 +12,19 @@ var sha256 = require('sha256');
 /**
  * GET / [user] - Get user object, return if current user matches or is admin
  * @param req.query.email - user email (required)
+ * @param req.query.curEmail - current user obj email (required)
+ * @param req.query.curAdmin - current user obj isAdmin bool (required)
  */
 router.get('/', perm.auth, function(req, res) {
```

Comment:
Actually, it seems like `perm.user` was built for this, it only allows the request if the user matches the one in the email. Could you replace `perm.auth` with `perm.user`, and make sure it doesn't break any other places that the `/api/user` route is used?